### ISSUE TITLE
Galaxy star rendering issue

### ISSUE THREAD

**Issue by micahnyc** [OpenSpace/OpenSpace#1596]
When zooming away from galaxy, stars in rendering become black and you cannot see the galaxy clearly. 

![image](https://user-images.githubusercontent.com/708535/118725280-bbc83080-b7fd-11eb-8ed7-dbd23e227e41.png)

Viewing the galaxy with stars turned off, you can see the galaxy clearly.

![image](https://user-images.githubusercontent.com/708535/118725296-c1257b00-b7fd-11eb-8d74-e6f496f86676.png)

**Comment by alexanderbock** [comment]
Is this a problem on the `master`?  I thought I had fixed that in 863a86194405a4b8ffa5de37401abb5fdcc10b80

**Comment by WeirdRubberDuck** [comment]
Yes, there still seems to be some weirdness going with the volume rendering on master. Although, it no longer becomes completely black 🤔 

https://user-images.githubusercontent.com/8808894/118790503-1e6d0b00-b896-11eb-9b09-4b93280685a5.mov

**Comment by Arohdin** [comment]
Related to rendering of the Stars as points for RenderableGalaxy.
Issue does not happen with "billboard" option.
